cloud_functions/gcs_event_based_ingest/Dockerfile.ci

```diff
@@ -1,6 +1,4 @@
 FROM python:3.8-slim
 COPY requirements.txt requirements-dev.txt ./
 RUN pip3 install -r requirements-dev.txt
-WORKDIR /ci
```

Comment:
cc: @danieldeleo FYI because we know cloud build we have the source cloned locally we can remove this step speeding up image build time significantly as the ci container doesn't need the source inside the container.

Reply:
I was wondering about this! Thanks for the fyi